src/masonite/commands/AuthCommand.py

```diff
@@ -19,13 +20,21 @@ def __init__(self, application):
         self.app = application
 
     def handle(self):
-        copy_tree(
-            self.get_template_path(),
-            views_path("auth"),
-        )
-        copy_tree(self.get_controllers_path(), controllers_path("auth"))
-
-        copy_tree(self.mailables_path(), mailables_path())
+        # dirs_exist_ok option is available in Python 3.8+
+        # https://docs.python.org/3/library/shutil.html#shutil.copytree
+        if sys.version_info.minor >= 8:
+            copytree(self.get_template_path(), views_path("auth"), dirs_exist_ok=True)
+            copytree(
+                self.get_controllers_path(),
+                controllers_path("auth"),
+                dirs_exist_ok=True,
+            )
+
+            copytree(self.mailables_path(), mailables_path(), dirs_exist_ok=True)
+        else:
+            copytree(self.get_template_path(), views_path("auth"))
+            copytree(self.get_controllers_path(), controllers_path("auth"))
+            copytree(self.mailables_path(), mailables_path())
```

Comment:
Why don't we need `dirs_exist_ok` here?

Reply:
Because the feature (and the argument) does not exist in Python < 3.8.
So for Python 3.7, we will loose the behaviour of automatically overriding the folders if they exist already !

Reply:
ooh ok I see whats going on here